src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private int size;
+    
+    @BeforeEach
+    void setUp() {
+        Storage.people.add(new User(123456789L, "Bob123", "111111111", 19));
+        Storage.people.add(new User(123456789L, "Alice123", "222222222", 22));
+        Storage.people.add(new User(123456789L, "John123", "333333333", 25));
+        size = Storage.people.size();
+    }
+    
+    @Test
+    void notContainsUserInStorage_Ok() throws ValidationException {

```

Comment:
please, follow naming convention from the task in all tests
![image](https://user-images.githubusercontent.com/11480043/230768273-03268479-394e-4548-9766-279b75c247a6.png)